src/DnDList.elm

```diff
@@ -388,6 +388,10 @@ and [sorting on drop](https://annaghi.github.io/dnd-list/config/operations-drop)
 
   - `InsertBefore`: The drag source item will be inserted before the drop target item.
 
+  - `InsertAround`: The drag source item will be inserted either before or after
+    the drop target item, depending on which half (left/right) of the drop target
+    item it was dropped on.
+
   - `Rotate`: The items between the drag source and the drop target items will be circularly shifted.
```

Comment:
What do you think about the vertical lists? Are you going to support those?

Reply:
Vertical lists - I totally forgot about them! Do you only distinguish them using the `Movement` type? If yes, we should probably do different x/y checks for each of the Movement types?

Grid of items - how's that represented? Is it still a List? Can you give me an example?

Reply:
Two examples for grid are here: `Config.Movement.FreeOnDrag` and  `Config.Movement.FreeOnDrop`